certified-connectors/Lex Power Sign/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "The API key is a bearer token. Place it in the Authorization header (Bearer act_xxxx)",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#00AED9",
+    "capabilities": [
+      "actions"
+    ],
+    "publisher": "Lex Persona",
```

Comment:
Hello @Lex-Olivier-M, 
Please update sections as mentioned below. 
    "publisher": "Name of company or organization.",
    "stackOwner": "Name of the owner of the service or product."
Please refer the template here - https://github.com/microsoft/PowerPlatformConnectors/blob/3ac7cb2ca3a494950d3a6810d2aeeb69f7ccc507/templates/certified-connectors/apiProperties.json

Reply:
Hello @vmanoharas. Thank you for your feedback. I regret but I do not understand your request. publisher and stackOwner have both the 'Lex Persona' value in my apiProperies.json. Lex Persona' is both the connector publisher and the stack owner (www.lex-persona.com). Could you please be more specific ? Regards.